src/main/java/seedu/duke/commands/ExchangeCommand.java

```diff
@@ -123,4 +124,28 @@ public BigDecimal parseAmount () throws InvalidNumberException {
             throw new InvalidNumberException();
         }
     }
+
+    /**
+    * Prints a message indicating if any required accounts are missing.
+    *
+    * @param ui the user interface to use for printing the message
+    * @param accounts the list of accounts to check for missing accounts
+    */
+    private void printMissingAccounts (Ui ui, AccountList accounts) {
+        try {
+            Forex exchangeRate = formatInput();
```

Comment:
This command is only called after input is already formatted. Why format the input again and have to catch the format input exception? The try-catch within try-catch should be avoided.
Since this is a private method, I suggest clarifying its use in javadocs to be only after a NoAccountException is met in the default code path, also storing the result of formatInput from the initial call